projects/sdk/src/lib/silo/Transfer.ts

```diff
@@ -0,0 +1,65 @@
+import { ContractTransaction } from "ethers";
+import { TokenValue } from "src/classes/TokenValue";
+import { BeanstalkSDK } from "../BeanstalkSDK";
+
+export class Transfer {
+  static sdk: BeanstalkSDK;
+
+  constructor(sdk: BeanstalkSDK) {
+    Transfer.sdk = sdk;
+  }
+
+  /**
+   * Initates a transfer of BEAN from the silo.
+   * @param amount The desired amount to transfer. Must be 0 < amount <= total deposits for token
+   * @param destinationAddress The destination address for the transfer
+   * @returns Promise of Transaction
+   */
+  async transfer(amount: TokenValue, destinationAddress: string): Promise<ContractTransaction> {
+    Transfer.sdk.debug("silo.transfer()", { amount, destinationAddress });
+
+    const SOURCE_BEAN_TOKEN = Transfer.sdk.tokens.BEAN;
```

Comment:
We want to be able to transfer any of the Silo tokens, ie the whitelisted tokens. This should be a parameter the user passes in, which we validate (error if trying to transfer a non-whitelisted token)

Reply:
OK I'll check the UI again. I thought it was only bean.

Reply:
I see 
```
BEAN
BEAN3CRV
urBEAN
urBEAN3CRV
```

Reply:
https://github.com/BeanstalkFarms/Beanstalk/pull/307/commits/62cd6dccf29437cf7eda4f2d8a0be936d397087e